### ISSUE TITLE
Must be easy to update

### ISSUE THREAD

**Issue by DocCyblade** [DocCyblade/tkl-rt#5]
Try and make it easy to update the copy of RT if needed.

**Comment by DocCyblade** [comment]
I am trying to install via package, however it could be hard to update. I am thinking we may need to do this via download. This way savvy users can upgrade. The TKL way would be to use TKLBAM from one server to the new and run upgrade scripts. Also packages are slow to be updated.

**Comment by JedMeister** [comment]
Yes packages are slow to be updated, but the flip-side of that is that they get auto security patches nightly. If you go with the packaged version then I would personally not bother supporting updates from upstream. It will just add headaches IMO.

I must say though I find the idea of 2 appliances; one using the Debian package, one using upstream source an interesting solution. I'm not sure how the core devs (Alon and Liraz) would feel about having 2 appliances of the same software though. I suspect that if the upstream version of RT is that much better than the Debian repo version then go upstream.

Having said that I don't think that this is wasted effort. It's a good place to start on TKLDev IMO and will allow you to have an appliance built much quicker.

**Comment by DocCyblade** [comment]
From a "get it done quickly" stand point, using the packages would be the way to go. As I said before they are way behind (packaged is 4.0.7 - updated Aug 20, 2012 ) (current is 4.0.22 updated Sept 4, 2014) thats a long time with no love! 

I don't know what the core devs thoughts or road map is for all the appliances out there now. Do they all get updated at the same time ( I assume this is the case since I have read they are in a long dev cycle right now) 

I thought however the goal of TKLDev was to get the community more involved (like I am doing now, and others out there) so that the core devs would not have so much to do. 

I may still release a 4.0.7 packaged version then a 4.0.21 version. Right now I am working on 4.0.21 just for fun. I think I will create two lines an (us or up stream) and a (pk or packaged)

**Comment by DocCyblade** [comment]
After digging I did find an updated version in back ports 
https://packages.debian.org/wheezy-backports/request-tracker4 currently at 4.0.19 (only two behind)

**Comment by JedMeister** [comment]
> I don't know what the core devs thoughts or road map is for all the appliances out there now. Do they all get updated at the same time ( I assume this is the case since I have read they are in a long dev cycle right now)

Historically all updated appliances have been released together. However there have been discussions around releasing bug fixed appliances and probably new (community developed) appliances out of band (i.e. not all together). Regardless we do hope to get the release schedule back on track... (It is way behind schedule at the moment...)

> I thought however the goal of TKLDev was to get the community more involved (like I am doing now, and others out there) so that the core devs would not have so much to do.

For sure! That's the plan! :)
